apps/admin-client/src/styles.css

```
@@ -0,0 +1,370 @@
+@import 'tailwindcss';
+@import 'tw-animate-css';
+
+@plugin '@tailwindcss/typography';
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)